test_scripts/RC/CLIMATE_RADIO/OnInteriorVehicleData/017_Success_flow_with_climateEnable.lua

```diff
@@ -0,0 +1,53 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0213-rc-radio-climate-parameter-update.md
+-- Description:
+-- Preconditions:
+-- 1) SDL got RC.GetCapabilities for CLIMATE module with ("climateEnableAvailable" = true) parameter from HMI
+-- 2) Mobile app subscribed on getting RC.OnInteriorVehicleData notification for CLIMATE module
+-- In case:
+-- 1) HMI sends RC.OnInteriorVehicleData notification ("climateEnable" = false) to SDL
+-- 2) HMI sends RC.OnInteriorVehicleData notification ("climateEnable" = true) to SDL
+-- SDL must:
+-- 1) sends OnInteriorVehicleData notification ("climateEnable" = false) to Mobile
+-- 2) sends OnInteriorVehicleData notification ("climateEnable" = true) to Mobile
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonRC = require('test_scripts/RC/commonRC')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local mType = "CLIMATE"
+local paramValues = {
+  false,
+  true
+}
+
+--[[ Local Functions ]]
+local function notificationProcessedSuccessfully(pValue)
+  function commonRC.getAnotherModuleControlData()
```

Comment:
I am not familiar with this syntax. Does this overwrite the definition of the common `getAnotherModuleControlData()`?

Reply:
@JackLivio Correct. When `notificationProcessedSuccessfully()` function is executed implementation of `commonRC.getAnotherModuleControlData()` will be overwritten. And new code will be used till the end of the script.